src/processes.c

```diff
@@ -864,6 +870,7 @@ static procstat_t *ps_read_vmem (int pid, procstat_t *ps)
 
 	ps->vmem_data = data * 1024;
 	ps->vmem_code = (exe + lib) * 1024;
+	ps->num_lwp = threads;
```

Comment:
Type mismatch: `unsigned long` vs. `unsigned long long`. Please change `data`, `exe`, `lib` and `threads` to `unsigned long` and change `strtoll` above to `strtoul`.